src/modules/app/constants/dimensions.js

```diff
@@ -1,5 +1,8 @@
 export default {
   smallViewport: `(max-width: 750px)`,
-  headerHeight: '85px',
-  headerHeightSmall: '55px'
+  minMaxWidth: `screen and (min-width: 1100px)`,
```

Comment:
Is this validated by styled-components?

Reply:
Hmm... the template-string itself not no, could be validated once its injected in styled-components-stylesheets 🤔 not sure though